task/packrat/src/main.rs

```diff
@@ -0,0 +1,140 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! VPD manipulation
```

Comment:
This isn't the best description of the task (and indeed appears copied from `task/vpd/src/main.rs`)

Reply:
It is indeed! Will fix.